tools/generator/src/Generator/XCSchemeInfo+BuildActionInfo.swift

```diff
@@ -54,17 +62,29 @@ extension XCSchemeInfo.BuildActionInfo {
         guard let buildAction = buildAction else {
             return nil
         }
+        let context = "creating a `BuildActionInfo`"
         let buildTargetInfos: [XCSchemeInfo.BuildTargetInfo] = try buildAction.targets
             .map { buildTarget in
                 let targetID = try targetIDsByLabel.value(
                     for: buildTarget.label,
-                    context: "creating a `BuildActionInfo`"
+                    context: context
                 )
                 return XCSchemeInfo.BuildTargetInfo(
                     targetInfo: try targetResolver.targetInfo(targetID: targetID),
                     buildFor: buildTarget.buildFor
                 )
             }
-        try self.init(targets: buildTargetInfos)
+
+        try self.init(targets: buildTargetInfos,
+                      preActions: buildAction.preActions.resolve(
+                          targetResolver: targetResolver,
+                          targetIDsByLabel: targetIDsByLabel,
+                          context: context
+                      ),
+                      postActions: buildAction.postActions.resolve(
+                          targetResolver: targetResolver,
+                          targetIDsByLabel: targetIDsByLabel,
+                          context: context
+                      ))
```

Comment:
nit: In this repo, function calls that extend to multiple lines put all of the arguments on separate lines. I think that this should look like this:

```swift
        try self.init(
            targets: buildTargetInfos,
            preActions: buildAction.preActions.resolve(
                targetResolver: targetResolver,
                targetIDsByLabel: targetIDsByLabel,
                context: context
            ),
            postActions: buildAction.postActions.resolve(
                targetResolver: targetResolver,
                targetIDsByLabel: targetIDsByLabel,
                context: context
            ),
        )
```